requirements.in

```diff
@@ -29,7 +29,7 @@ pyproj==3.4.0
 # PaaS
 awscli-cwlogs>=1.4,<1.5
 itsdangerous==2.1.2
-emergency-alerts-utils @ git+https://github.com/alphagov/emergency-alerts-utils.git@60.0.0-alpha
+emergency-alerts-utils @ git+https://github.com/alphagov/emergency-alerts-utils.git
```

Comment:
Justensuring that this change was made by running `make freeze-requirements`

Reply:
Good point... the requirements.in file is manually adjusted. `make freeze-requirements` builds the requirements.txt file - this has been done below.

Note: The ordering of packages in requirements.txt may be due to using an updated version of `pip`